resources/views/test-unit-notification.blade.php

```diff
@@ -0,0 +1,19 @@
+/* @@covers \{{ $covered }} */
+
+use {{ $classFqcn }};
+
+it('formats the mail', function (): void {
+    $notification = new {{ $classBasename }}();
+
+    $this->assertEquals(['mail'], $notification->via());
+
+    $mail = $notification->toMail();
+
+    $this->assertEquals('', $mail->subject);
+    $this->assertEquals('', $mail->greeting);
+    $this->assertEquals([''], $mail->introLines);
+    $this->assertEquals('', $mail->actionText);
+    $this->assertEquals('', $mail->actionUrl);
+    $this->assertEquals([''], $mail->outroLines);
+    $this->assertEquals('', $mail->salutation);
```

Comment:
use pest expectation api

Reply:
Done in 8f1b5553e0d6d7d319d8f7c12310eb939f4dce30